geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -70,11 +70,27 @@ <h2 class="sr-only">
                 {% endif %}
             </dl>
             {# secondary metadata #}
-            {% if document.languages.exists or document.secondary_languages.exists %}
+            {% if document.languages.exists or document.secondary_languages.exists or document.doc_date_original or document.doc_date_standard %}
                 <dl class="metadata secondary">
-                    {# NOTE: placeholder to populate translation string #}
-                    {# Translators: label for date of this document, if known #}
-                    <!-- {% translate "Document Date" %} -->
+                    {% if document.doc_date_original or document.doc_date_standard %}
```

Comment:
revise to check the `document_date` method I'm suggesting you write (will be more extensible if/when we add other date options)